### DISCUSSION TITLE
Ukrainian translation

### DISCUSSION THREAD

**Discussion by sunweaver** [kimai/kimai#3612] [General]
### Describe the problem

Kimai2 has not been translated to uk_UA, yet.

### Describe the solution you'd like

In the company [Fre(i)e Software GmbH], one of our employees is fluent in Ukrainian and Russian and he will start on adding those translations as a FLOSS contribution to your project. We have been using Kimai2 for a while now and it is awesome. This is our way of sending a big THANK YOU!

**Comment by kevinpapst** [comment]
Hi @sunweaver - awesome, great offer, thank you!

He should register at Weblate and then he can just start adding translations here:

Ukrainian: https://hosted.weblate.org/languages/uk/kimai/
Russian: https://hosted.weblate.org/languages/ru/kimai/
